### DISCUSSION TITLE
Can get real time data from tradingview

### DISCUSSION THREAD

**Discussion by over-infinity** [Mathieu2301/TradingView-API#72] [Q&A]
Hello,
At first, thank you for your work and this open-source utility.
I want to know at first is it possible to get real time data from tradingview and my second question is that is there any visual example of using real time in tradingview widget or not?
Thanks al lot

**Comment by Mathieu2301** [comment]
> I want to know at first is it possible to get real time data

This is literally the description of the library.

> is there any visual example of using real time in tradingview widget or not?

I don't understand your request. You can find some examples [here](https://github.com/Mathieu2301/TradingView-API/tree/main/examples).
